CI Flaky test back

## Bug Report

https://github.com/suitenumerique/docs/actions/runs/20711339733/job/59452356166?pr=1764

`FAILED core/tests/test_api_users.py::test_api_users_list_query_short_queries - assert 3 == 2`

<img width="960" height="280" alt="Image" src="https://github.com/user-attachments/assets/f054906c-0cf7-47b9-a71c-aa89e3ad08e8" />

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,10 @@ and this project adheres to
 - ✨(backend) add documents/all endpoint with descendants #1553
 - ✅(export) add PDF regression tests #1762
 
+### Fixed 
+
+- ✅(backend) reduce flakiness on backend test #1769
+
 ## [4.3.0] - 2026-01-05
 
 ### Added

--- a/src/backend/core/tests/test_api_users.py
+++ b/src/backend/core/tests/test_api_users.py
@@ -311,7 +311,7 @@ def test_api_users_list_query_short_queries():
     """
     Queries shorter than 5 characters should return an empty result set.
     """
-    user = factories.UserFactory(email="paul@example.com")
+    user = factories.UserFactory(email="paul@example.com", full_name="Paul")
     client = APIClient()
     client.force_login(user)
 

